src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentInfo.cs

```diff
@@ -0,0 +1,260 @@
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>>
+        /// <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public ComponentInfo(string componentId)
+        {
+            _componentId = componentId;
+            IntPtr infoHandle = IntPtr.Zero;
+            Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoCreate(_componentId, out infoHandle);
+            if (err != Interop.ComponentManager.ErrorCode.None)
+            {
+                throw ComponentManager.ComponentManagerErrorFactory.GetException(err, "Failed to create the ComponentInfo.");
+            }
+            _infoHandle = infoHandle;
+        }
+
+        /// <summary>
+        /// Destructor of the class.
+        /// </summary>
+        ~ComponentInfo()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Gets the component ID.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string ComponentId
+        {
+            get
+            {
+                if (!string.IsNullOrEmpty(_componentId))
+                    return _componentId;
+
+                string compId = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetComponentId(_infoHandle, out compId);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the ComponentId. err = " + err);
+                }
+                _componentId = compId;
+
+                return _componentId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the application ID of the component.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string ApplicationId
+        {
+            get
+            {
+                string appId = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetAppId(_infoHandle, out appId);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the ApplicationId of " + _componentId + ". err = " + err);
+                }
+
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the type of the component.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ComponentType Type
```

Comment:
```suggestion
        public ComponentType ComponentType
```
It confused with GetType()